index.js

```diff
@@ -0,0 +1,88 @@
+import "./style.css";
+import Task from "./class.js";
+
+const itemsDisplay = document.getElementById("container");
+const itemValue = document.getElementById("task");
+const addItemBtn = document.getElementById("add");
+const eraseAll = document.querySelector(".eraser");
+
+const tasks = new Task();
+
+const display = () => {
+  itemsDisplay.innerHTML = "";
+
+  for (let i = 0; i < tasks.todo.length; i += 1) {
```

Comment:
Possibly also think about how to implement this using a forEach loop instead of a normal for loop. Take a look at [This link](https://medium.com/@abustamam/for-loops-vs-foreach-in-javascript-7a977278a39e) for more ideas and clarity